### ISSUE TITLE
Quasar CLI electron and mobile apps

### ISSUE THREAD

**Issue by garyrob** [elimintz/justpy#233]
Because justpy provides an interface to Quasar, I've been reading up on that. I note that with Quasar CLI: 

> The difference between building a SPA, PWA, Mobile App or an Electron App is simply determined by the “mode” parameter in “quasar dev” and “quasar build” commands. [https://v0-15.quasar-framework.org/guide/electron-configuring-electron.html]

I'm wondering whether it would be feasible for a justpy app to take advantage of this ability either now or in the future. Seems like it would be amazing to be able to leverage justpy to enable Python to deliver all those different kinds of apps with the same codebase. (I would expect there to be differences to target each environment, as with any responsive app, of course, but reusing most code.)

**Comment by elimintz** [comment]
Yes, that would be amazing. It is in my plans longer term to empower JustPy in such manner.

**Comment by garyrob** [comment]
Glad to hear that!

**Comment by garyrob** [comment]
Definite +1 for that ability.

**Comment by qiuwei** [comment]
This will be super cool!
